src/app/hero-detail/hero-detail.component.css

```diff
@@ -0,0 +1,25 @@
+    /* HeroDetailComponent's private CSS styles */
+    label {
```

Comment:
I think everything in this file should be [globally applied styles](https://angular.io/guide/component-styles#external-and-global-style-files).

At the moment the styles for base elements like a button, label or an input is introducing more code than necessary and little inconsistencies.

It'd be good to provide a set of base/default styles that these base elements inherit then they can be overridden where necessary.